wails-frontend/src/pages/importMethods.tsx

```diff
@@ -13,18 +14,28 @@ const ImportMethods = () => {
   const handlePkey = () => navigate('/import-pkey', { state: { req } });
 
   return (
-    <section>
-      <div>{req.Msg}</div>
-      <div className="baseline">{baselineStr}</div>
-      <div className="flex flex-col">
-        <button className="btn" onClick={handleYml}>
-          I have a .yml file
-        </button>
-        <button className="btn" onClick={handlePkey}>
-          I have a private key
-        </button>
+    <div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
+      <div className="w-1/4  max-w-xs  min-w-fit">
```

Comment:
Same here about **max-w-sx**. Please, double check if we need it!

Reply:
see first comment !